animations.md

```diff
@@ -1,311 +1,377 @@
-itself.
-
-The animation controller can be given a lower and upper bound to
-animate between, and a duration.
-
-In the simple case (using `forward()`, `reverse()`, `play()`, or
-`resume()`), the animation controller simply does a linear
-interpolation from the lower bound to the upper bound (or vice versa,
-for the reverse direction) over the given duration.
-
-When using `repeat()`, the animation controller uses a linear
-interpolation between the given bounds over the given duration, but
-does not stop.
-
-When using `animateTo()`, the animation controller does a linear
-interpolation over the given duration from the current value to the
-given target. If no duration is given to the method, the default
-duration of the controller and the range described by the controller's
-lower bound and upper bound is used to determine the velocity of the
-animation.
-
-When using `fling()`, a `Force` is used to create a specific
-simulation which is then used to drive the controller.
-
-When using `animateWith()`, the given simulation is used to drive the
-controller.
-
-These methods all return the future that the `Ticker` provides and
-which will resolve when the controller next stops or changes
-simulation.
-
-### Attaching animatables to animations
-
-Passing an `Animation<double>` (the new parent) to an `Animatable`'s
-`animate()` method creates a new `Animation` subclass that acts like
-the `Animatable` but is driven from the given parent.
+You can use animations to make the user experience on your mobile app feel more
+intuitive, dynamic, and polished. Animations are especially useful when the
+screen changes state, such as when content loads or when new actions become
+available. This how-to guide shows some common ways you can add animation
+effects to your apps using Flutter’s animation system.
+
+* [Overview of the Flutter animation system](#overview-of-the-flutter-animation-system)
+* [Rendering your animation](#rendering-your-animation)
+* [Disposing your animation](#disposing-your-animation)
+
+## Overview of the Flutter animation system
+
+Flutter has two separate animation libraries:
+
+  * <code>[flutter/animation](http://docs.flutter.io/flutter/animation/animation-library.html)</code>:
+  Provides APIs to create animation effects in user interfaces (UIs) that are
+built on the <code>[StatelessWidget](http://docs.flutter.io/flutter/material/StatelessWidget-class.html)</code>
+and <code>[StatefulWidget](http://docs.flutter.io/flutter/widgets/StatefulWidget-class.html)</code>
+classes.
+  * <code>[flutter\_sprites/flutter\_sprites](http://docs.flutter.io/flutter_sprites/flutter_sprites/flutter_sprites-library.html)</code>:
+  Provides APIs for game development.
+
+The rest of this documentation covers how to use the APIs from the
+<code>[flutter/animation](http://docs.flutter.io/flutter/animation/animation-library.html)</code>
+library.
+
+Animations in Flutter are encapsulated as
```

Comment:
You never say what Animations are. The reader has no context to understand why you are telling them about Value and AnimationStatus.